src/l500/l500-options.cpp

```diff
@@ -707,7 +707,7 @@ namespace librealsense
             {
                 for (auto opt : _hw_options)
                 {
-                    bool success;
+                    bool success = true;
```

Comment:
We can merge it but please avoid changes and effort in L500 specific as it will soon be removed from the code.
We need to focus on SDK internals + D400/D500 only.

Thanks

Reply:
You know what, lets drop this change and make Eran life easier , he is currently working on removing this so the less conflicts the better.

Reply:
May we remove a specific [commit](https://github.com/IntelRealSense/librealsense/pull/11738/commits/7e978201225d7ef5955e5768b0fa3d18fc20e110) from the PR?

Reply:
Yes, it is possible to cherry-pick changes but I don't think it is needed, it also creates a new commit.
You can revert the change in a new commit and when merging with `development` the file will not change.